src/main/java/org/example/DBController.java

```diff
@@ -128,6 +129,142 @@ public boolean updateCustomerFunds(int customerId, int newAmount) {  //void
         return false;
     }
 
+    public void createTransactionAudit(int senderId, String senderName, int moneyInCents, int getterId, String getterName) throws SQLException {
+        String sql = "INSERT INTO transactions_audit (sender_id, sender_nickname, amount_sent, receiver_id, receiver_nickname) " + "VALUES (?, ?, ?, ?, ?)";
+
+        PreparedStatement ps = connection.prepareStatement(sql,
+                Statement.RETURN_GENERATED_KEYS);
+        {
+
+            ps.setInt(1, senderId);
+            ps.setString(2, senderName);
+            ps.setInt(3, moneyInCents);
+            ps.setInt(4, getterId);
+            ps.setString(5, getterName);
+
+            int rows = ps.executeUpdate();                          // returns 1

```

Comment:
Is it possible to do it like this?
```suggestion
            int rowsInserted = preparedStatement
                .setInt(1, senderId)
                .setString(2, senderName)
                .setInt(3, moneyInCents)
                .setInt(4, getterId)
                .setString(5, getterName)
                .executeUpdate()
                ;
```
This is called "fluent setters". See https://medium.com/miro-engineering/fluent-setter-breaking-the-convention-33ce3433126e. It's not a rule, but it generally is a very common stylistic pattern in code.